### PR TITLE
Video details playlist button

### LINKED ISSUE #2124: Video tile details improvements
**Changelog:** 
• Added new "Type" property with values: "Video" and "Playlist" so we are able to support Playlist tiles with that components (added tertiary button with "View playlist details" label)

**Figma link:**
https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-Components?node-id=3630%3A112904

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #2124 

**Design**
https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-components?node-id=3630%3A112904

### CODE REVIEW

**Review by eldiegod — APPROVED** [review]
Approving, button doesn't seem to match the design but it's because the latest button update is not in this branch

File: packages/atlas/src/components/_video/VideoTileDetails/VideoTileDetails.tsx [hunk 1]
```diff
@@ -98,15 +103,23 @@ export const VideoTileDetails: React.FC<VideoTileDetailsProps> = ({
...
+                      {videoSubTitle
+                        ? videoSubTitle
+                        : createdAt && (
+                            <>
+                              {formatVideoDate(createdAt)} • <Views>{formatVideoViews(views || 0)}</Views>
+                            </>
+                          )}
+                    </Text>
+                  ) : (
+                    <PlaylistButton onClick={onPlaylistDetailsClick} variant="tertiary" size="small" textOnly>
```
Comment (kdembler) [inline packages/atlas/src/components/_video/VideoTileDetails/VideoTileDetails.tsx]: I think this should be a regular link, not a button

**Review by toiletgranny — APPROVED** [review]
If my understanding is correct that the scope of this PR is just adding the "View playlist details" button to the `VideoTileDetails` component, then — no doubt about — this has been accomplished. :D

The button looks different, but I'm assuming this is because of the button update not being in this brunch, as @eldiegod is saying.

**Comment by kdembler** [comment]
Just to make sure, the goal wasn't to introduce a button but rather create a `Playlist` variant of the video tile details component. If the button is all that's needed, I'll leave that up you :D

**Pull request merged**